docs/quickstart/infrastructure.md

```diff
@@ -7,50 +7,82 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/quickstart/
 
 # Infrastructure monitoring with Netdata
 
-Together, the Netdata Agent and Netdata Cloud create a powerful, infinitely-scalable infrastructure monitoring solution.
+[Netdata Cloud](https://app.netdata.cloud) provides scalable infrastructure monitoring for any number of distributed
+nodes running the Netdata Agent.
```

Comment:
Can we add a small mention that says a node is a machine/vm/whatever? Just so that the user must not assume that.

Reply:
Sure. We've been using **node** as the encompassing term for a long time now; see the [glossary](https://learn.netdata.cloud/docs/cloud/faq-glossary#glossary-of-netdata-cloud-terms).

Reply:
You are right.